config-windows.md

```diff
@@ -82,6 +82,8 @@ The following parameters can be specified (mutually exclusive):
 * **`count`** *(uint64, OPTIONAL)* - specifies the number of CPUs available to the container. It represents the fraction of the configured processor `count` in a container in relation to the processors available in the host. The fraction ultimately determines the portion of processor cycles that the threads in a container can use during each scheduling interval, as the number of cycles per 10,000 cycles.
 * **`shares`** *(uint16, OPTIONAL)* - limits the share of processor time given to the container relative to other workloads on the processor. The processor `shares` (`weight` at the platform level) is a value between 0 and 10,000.
 * **`maximum`** *(uint16, OPTIONAL)* - determines the portion of processor cycles that the threads in a container can use during each scheduling interval, as the number of cycles per 10,000 cycles. Set processor `maximum` to a percentage times 100.
+* **`affinity`** *(object, OPTIONAL)* - specifies the set of CPU to affinitize for this container.
+* **`affinityPreferredNumaNodes`** *(uint32, OPTIONAL)* - specifies referred set of numa node numbers to affinitize for this container.
```

Comment:
I think someone using this would naturally ask "how do I know which node is 0, 1, etc?". How would they determine this? Is there an API we can tell them to use that will match the indexing expected here?